pkg/apis/core/v1beta1/constants/types_constants.go

```diff
@@ -284,6 +284,8 @@ const (
 	// Note that this annotation is alpha and can be removed anytime without further notice. Only use it if you know
 	// what you do.
 	ShootAlphaControlPlaneScaleDownDisabled = "alpha.control-plane.scaling.shoot.gardener.cloud/scale-down-disabled"
+
+	// ShootAlphaControlPlaneHighAvailability TODO(shreyas-s-rao): deprecate HA annotation with the stable release of zonal clusters feature
```

Comment:
```suggestion
	// TODO(shreyas-s-rao): Deprecate HA annotation with the stable release of zonal clusters feature.
```

Also, move this to the last comment line